### PR TITLE
Add missing jest-validate to dependencies

### PR DESCRIPTION
**Pull request by lydell**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,11 +23,11 @@
     "flow-parser": "0.38.0",
     "get-stdin": "5.0.1",
     "glob": "7.1.1",
+    "jest-validate": "18.2.0",
```
Comment (xtuc) [inline package.json]: This is part of the development tools. Why would you move it?

Comment (hzoo) [inline package.json]: Got added to the src https://github.com/jlongster/prettier/blob/66fe2f78fa99789e9f784149381ff60265754481/src/options.js#L3

Comment (xtuc) [inline package.json]: OK, sorry :+1: nevermind.

**Comment by vjeux** [comment]
Just published 0.13.1 with this fix.

**Comment by vjeux** [comment]
We should probably add an end-to-end test that tries the published version via `npm pack`

**Pull request merged**